kwok/README.md

```diff
@@ -5,14 +5,25 @@ Before using the kwok provider, make sure that you don't have an installed versi
 ## Requirements
 - Have an image repository that you can build, push, and pull images from.
   - For an example on how to set up an image repository refer to [karpenter.sh](https://karpenter.sh/docs/contributing/development-guide/#environment-specific-setup)
+  - When working with a Kind cluster, run
+  ```bash
+  export KWOK_REPO=kind.local
```

Comment:
you should also need `KIND_CLUSTER_NAME` set as well, no?

Reply:
I didn't set this when running locally. I'll retest with a new shell.

Reply:
Wasn't necessary.